src/components/Pagination.jsx

```diff
@@ -2,35 +2,38 @@ import { useAtom } from "jotai";
 import { ArrowIcon } from "../assets/icons";
 import { currentBatchSize } from "../utils/jotai";
 import Dropdown from "./Dropdown";
+import { notify, Type } from "./Notifier";
 
 export const Pagination = ({
     loadNextBatch,
     loadPrevBatch
 }) => {
     const [batchSize, setBatchSize] = useAtom(currentBatchSize);
-    const onClickHandler = (size) => {
-        setBatchSize(size)
-    }
 
     return (
         <div className="w-full p-2 flex justify-end items-center">
             <div
                 className="w-6 h-6 cursor-pointer hover:scale-125 transition active:scale-100"
-                onClick={() => loadPrevBatch()}>
+                onClick={async () => {
+                    if (await loadPrevBatch() === false) notify(Type.error, 'No more data to load')
```

Comment:
Technically not required, but it's good formatting in js/jsx to put a semicolon here,